client/public/index.html

```diff
@@ -22,6 +22,7 @@
 
     <script src="./jquery-3.3.1.min.js"></script>
     <script src="./bootstrap.min.js"></script>
+    <script src="https://unpkg.com/react-measure/dist/react-measure.js"></script>
```

Comment:
@donia98 I think you can remove this since you put react-measure into package.json
Whenever you do use unpkg, it's probably safest to specify the package version in the src, i.e. `src="https://unpkg.com/react-measure@2.1.3/dist/react-measure.js"`.  Otherwise, unpkg will give the users the `@latest` version of the package, which means one day react-measure could release a new version with changes that break your app, and users loading the app on that day would get the new (breaking) version - without you even doing a code push.